modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java

```diff
@@ -128,13 +128,13 @@ protected void doProcessAction(
 				Date accessTokenExpirationDate =
 					oAuth2Authorization.getAccessTokenExpirationDate();
 
-				int daysBetween = DateUtil.getDaysBetween(
+				int daysToExpireAccessToken = DateUtil.getDaysBetween(
```

Comment:
Are you sure the old old name is not following SF conventions more?
I think convention suggests it should either be called "daysBetween" or simply "date" (named after the type.

Reply:
Updated https://github.com/liferay-appsec/liferay-portal/pull/1977/commits/20c84a12b456ff0fc4cbe1a03e058e207ef00790